inc/notificationtargetentity.class.php

```diff
@@ -57,11 +59,41 @@ public function addDataForTemplate($event, $options = []) {
       );
       $data = $req->current();
       $this->data['##credit.quantity_remaining##'] = (int)$this->obj->getField('quantity') - (int)$data['consumed_total'];
+      $this->data['##credit.quantity_consumed##'] = (int)$data['consumed_total'];
+
+      $req = $DB->request(
+         [
+            'SELECT' => 'name',
+            'FROM'   => 'glpi_entities',
+            'WHERE'  => [
+               'id' => $this->obj->getField('entities_id'),
+            ]
+         ]
+      );
+      $data = $req->current();
+      $this->data['##credit.entity##'] = $data['name'];
+
+      $req = $DB->request(
+         [
+            'SELECT' => 'name',
+            'FROM'   => 'glpi_plugin_credit_types',
+            'WHERE'  => [
+               'id' => $this->obj->getField('plugin_credit_types_id'),
+            ]
+         ]
+      );
+      $data = $req->current();
```

Comment:
Since we are loading only one item, we could use directly the `::getByID()` method

Reply:
If we just want the name of a dropdown, the `Dropdown::getDropdownName()` method is even better.